packages/ibm-products/src/components/BigNumber/BigNumber.figma.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { BigNumber } from './BigNumber';
+import figma from '@figma/code-connect';
+
+/**
+ * -- This file was auto-generated by Code Connect --
+ * `props` includes a mapping from Figma properties and variants to
+ * suggested values. You should update this to match the props of your
+ * code component, and update the `example` function to return the
+ * code example you'd like to see in Figma
+ */
+
+figma.connect(
+  BigNumber,
+  'https://www.figma.com/design/0F9dKH2abAd7gSfvnacfWf/-v11--IBM-Products-%E2%80%93-Carbon-Design-System?node-id=32105%3A811',
+  {
+    props: {
+      trending: figma.boolean('Trending'),
+      iconButton: figma.boolean('Edit button'),
+      total: figma.boolean('Total'),
+      label: figma.boolean('Label'),
```

Comment:
One more small suggestion here, can we incorporate the new property in Figma `Label text`

```tsx
label: figma.boolean('Label', {
  true: figma.string('Label text'),
}),
```